lib/api/controllers/remoteActions/swagger.js

```diff
@@ -0,0 +1,137 @@
+var
+  fs = require('fs'),
+  _ = require('lodash'),
+  path = require('path'),
+  InternalError = require('kuzzle-common-objects').Errors.internalError,
+  q = require('q'),
+  stringify = require('json-stable-stringify'),
+  jsonToYaml = require('json2yaml'),  
+  generateSwagger,
+  writeFiles;
+
+generateSwagger = (routes, apiVersion) => {
+  var swagger;
+
+  swagger = {
+    basePath: '/api',
+    consumes: [
+      'application/json'
+    ],
+    host: 'sandbox.kuzzle.io:7511',
```

Comment:
why not ? :) the goal of this url is to allow people to test the routes, so a sandbox looks a good idea to me